string type
string name
string id
string loctype
---
string id
